.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.7
```

Comment:
I'd be careful - as a library we should aim for wider support, but on the other hand only we use it, on the gripping hand, we are only on 2.5 on Lambda runtime